README.md

```diff
@@ -100,18 +100,23 @@ namespace similar to the following.
     istio-ingress-174722661-3fv5x        1/1       Running   0          20h
     istio-pilot-1557643696-zt8jg         1/1       Running   0          20h
 
-You should see a `dikastes-node` pod for each host in your cluster.  Dikastes is a Calico component that computes
-authorization policy for the Istio proxies on each host.  It is a separate component in this technical preview, but our
-plan is to integrate it into the `calico-node` pod in future versions.
-
 Note that Istio Mixer is not included in this demo because it is not required.  You can add a
 Mixer deployment and [use it for telemetry or additional authorization checks](#can-i-use-dikastes-with-istio-mixer?).
 
-Finally, add the Sidecar Injector, which automatically adds the Istio Proxy sidecar to each deployment you add to your
-cluster.
+Finally, add the Sidecar Injector, which automatically adds `Istio Proxy and Calico Dikastes` sidecar to each
+deployment you add to your cluster.
 
     kubectl apply -f config/install/20-istio-sidecar-injector.yaml
-    
+
+Dikastes is a Calico component that computes authorization policy for the Istio proxies on each host. Calico
+utilizes [Istio's automatic sidecar injection](https://istio.io/docs/setup/kubernetes/sidecar-injection.html#automatic-sidecar-injection)
+to inject Dikastes container into a pod at pod creation time. Run the following command if you want to enable automatic
+sidecar injection for the namespace of your deployment.
+
+```
+kubectl label namespace <your namespace name> istio-injection=enabled
+```
```

Comment:
may be add the specific example for the `demo` namespace. So someone following the instructions here can just cut-copy-paste

Reply:
The `demo` namespace is just for my special setup. This guide does not use `demo` namespace so user has to do `copy-paste-modify`.